pkg/plugin/v3/api.go

```diff
@@ -40,7 +40,7 @@ import (
 // KbDeclarativePatternVersion is the sigs.k8s.io/kubebuilder-declarative-pattern version
 // (used only to gen api with --pattern=addon)
 // TODO: remove this when a better solution for using addons is implemented.
-const KbDeclarativePatternVersion = "v0.0.0-20200522144838-848d48e5b073"
+const KbDeclarativePatternVersion = "1cbf859290cab81ae8e73fc5caebe792280175d1"
```

Comment:
barring that this is just for the alpha setup, does KB declarative patterns plan to institute release/versioning with semver?

Reply:
Great question. @camilamacedo86  do you know?

Reply:
I do not think so. However, I raised an issue there to request it for the authors. See: https://github.com/kubernetes-sigs/kubebuilder-declarative-pattern/issues/130 Let's see.